samples/excel/tables/format-table.yaml

```diff
@@ -0,0 +1,194 @@
+id: ''
+gist: ''
+source: EXCEL
+author: Microsoft
+name: Format table
+description: Format a table
+script:
+  content: |+
+    $('#create-table').click(createTable);
```

Comment:
Rename create-table to setup so it is clear that this is not the core concept demonstrated in the snippet.